spamostack/keeper.py

```diff
@@ -119,6 +120,10 @@ def get(self, client_name, resource_name, param=None, func=None,
                     else:
                         probe = getattr(el, param)(*args, **kwargs)
 
+                    if (resource_name == "flavors"
```

Comment:
Possibly yes :)
But if we hardcode some parameters, this will be wrong too (from my POV). 
As possible solution, we can save current state to db, and duplicate this info for more safety. Or, we can change mechanism with assigning custom id, like `spamo-{iteration_number}`, and keep resource, if id doesn't start with `spamo`